ank/src/cli_commands/get_workloads.rs

```diff
@@ -139,7 +150,7 @@ mod tests {
         mock_server_connection
             .expect_get_complete_state()
             .with(eq(vec![]))
-            .return_once(|_| Ok(Box::new(test_data)));
+            .return_once(|_| Ok((ank_base::CompleteState::from(test_data)).into()));
```

Comment:
Maybe we can directly generate a proro (ank_base) complete state for test_data and use in instead of making 2 subsequent conversions.